aws-xray-auto-instrumentation-agent-installer/src/main/java/com/amazonaws/xray/agent/XRayAgentInstaller.java

```diff
@@ -60,11 +72,10 @@ public static void install(String bootstrapJarPath, String runtimeJarPath, Strin
         File runtimeJar = new File(runtimeJarPath);
         try {
             addURL(Injector.class.getClassLoader(), runtimeJar.toURI().toURL());
+            Injector.loadAgent(instrumentationProxy, bootstrapJarPath, agentArgs);
         } catch (Throwable e) {
-            ; // TO throw an exception or not...
+            System.out.println("Failed to add runtime agent loader to classpath, aborting instrumentation");
```

Comment:
Have we tested if System `println` error message is visible to customer in Lambda? How about using Log here?

Reply:
`System.out.println` is visible in Lambda according to the [docs](https://docs.aws.amazon.com/lambda/latest/dg/java-logging.html). We do not use any loggers since this code would be running (and therefore loading classes) before the agent's premain. The agent can throw errors if you load a class and then DiSCo attempts to modify its bytecode, so we try to load as few classes as possible before premain.